src/components/scene/stats.js

```diff
@@ -11,6 +11,11 @@ var AFrameStats = window.aframeStats;
  * Stats appended to document.body by RStats.
  */
 module.exports.Component = registerComponent('stats', {
+  schema: {
+    type: 'boolean',
```

Comment:
Bug for stats is #1619, So removed  `type: 'boolean'` from stats component and visible tested

- `<a-scene stats></a-scene>` OK
- `<a-entity visible geometry="primitive: box; depth: 1; height: 1; width: 1;" ></a-entity>` OK
- and both hide / show when toggling `true|false`